### DISCUSSION TITLE
DroneCAN Support for rddrone-bms772

### DISCUSSION THREAD

**Discussion by MallikarjunSE** [NXP-Robotics/RDDRONE-BMS772#7] [Q&A]
Hello, 
Is there any plans to implement DroneCAN on these boards? Going forward Ardupilot and Px4 will use DroneCAN instead of UAVCANv1. 
https://dronecan.github.io/ 
Can someone share link to UAVCANv0 implementation for this board? 
Thank you!

**Comment by cisvanmierlo** [comment]
Hello @MallikarjunSE ,

UAVCANv0 has been ported to the S32K platform before but not for this specific BMS772 board.
We did try to upstream our S32K1 libuavcan driver but that didn't came through see
https://github.com/UAVCAN/libuavcan/pull/303

We intent to revive that effort and add CAN FD support and upstream this to DroneCAN.
Once we have S32K1 support in DroneCAN we could take a look at implementing DroneCAN on the BMS772.

**Reply by alexscheck** [reply]
Hi, 
    Has there been any progress made on this? Thanks!

**Reply by igalloway** [reply]
There are pull requests for the BMS standard in DroneCAN, but so far that team has been slow in responding.  We're trying to escalate.

**Reply by igalloway** [reply]
https://github.com/dronecan/DSDL/pull/7#issuecomment-1385685872
@alexscheck  - if you help make a comment or UP-VOTE there it might help?
